sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -366,6 +368,51 @@ def test_model_pipeline_same_as_normalize_true(LinearModel, params):
     assert_allclose(y_pred_normalize, y_pred_standardize)
 
 
+# FIXME: 'normalize' to be removed in 1.2
+@pytest.mark.filterwarnings("ignore:'normalize' was deprecated")
+@pytest.mark.parametrize(
+    "estimator, params, is_sparse",
+    [(LinearRegression, {}, True),
+     (LinearRegression, {}, False)
+     ]
```

Comment:
```suggestion
     (LinearRegression, {}, False)]
```

Reply:
Is it to prepare for other deprecation? Maybe we want to move to a `test_common.py` this test then because it will be run across different estimator coming from different submodules.

Reply:
It is only going to be used by linear models and my understanding of `test_common.py` is that it is more general. Please let me know if I am mistaken, then I will update